src/VisualStudio/Core/Def/TableDataSource/TaskList/MiscellaneousTaskListTableControlEventProcessorProvider.cs

```diff
@@ -1,29 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
task list for misc workspace hasn't worked in forever.  this is because task list is based on incremental analyzers running, and those don't run on misc workspace *at all*.